Detectors/TPC/simulation/include/TPCSimulation/Digitizer.h

```diff
@@ -71,7 +71,7 @@ class Digitizer {
     /// Steer conversion of points to digits
     /// \param points Container with TPC points
     /// \return digits container
-    DigitContainer *Process(TClonesArray *points);
+    DigitContainer *Process(const std::vector<o2::TPC::LinkableHitGroup>* hits);
```

Comment:
Why not pass by const ref? Can this be a nullptr?

Reply:
This change could be done (but I wanted to change the underlying type first)